src/Model/FieldType/DateTimeField.ts

```diff
@@ -6,8 +6,8 @@ export const DateTimeField: IFieldType<Date> = {
   defaultValidationRules: { datetime: true },
   defaultValue: null,
   isValidType: (value) => value == null || isDate(value),
-  transform: (date: Date) => date != null ? date.toISOString() : null,
-  normalize: (value: string | any): Date => (
+  transform: async (date: Date) => date != null ? date.toISOString() : null,
```

Comment:
Same thing here.  Again, distinctly possible that I'm thinking about it wrong.

Reply:
See above